### DISCUSSION TITLE
Disable Auto Renew

### DISCUSSION THREAD

**Discussion by wcmgto** [w2c/letsencrypt-esxi#35] [General]
What is the best/easiest way to disable the auto renew feature? I have this running with a vCenter managed cluster with HA and the cert renewal doesn't play nice with HA. I'd like to turn off auto renew and manually renew as needed to see if that helps.

**Comment by Churro** [comment]
You could edit `/var/spool/cron/crontabs/root` on that host and simply remove the cronjob for auto-renewal.

**Reply by wcmgto** [reply]
It looks like editing the contab directly is a little convoluted. Is it possible to do this with the RENEW_DAYS variable in the renew.cfg file? Like, RENEW_DAYS=0

**Reply by Churro** [reply]
The script multiplies `RENEW_DAYS` with 86400, so not sure what would happen in case it is 0. If it doesn't work with 0, 1 should should be a good alternative.
